tutorials/misc/pausing_games.rst

```diff
@@ -50,6 +50,15 @@ property in a node:
 
 .. image:: img/pausemode.png
 
+You can achieve the same result in code:
+
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
```

Comment:
There's a trimming space at the beginning of line 57. Is that a problem @akien-mga ?

Reply:
The space is correct here.